swagger.json

```diff
@@ -389,6 +393,47 @@
                     }
                 }
             }
+        },
+        "/facility/company": {
+            "post": {
+                "description": "creates a new facility for a specific company",
+                "summary": "Enables a company's Travel Admin to create a new facility",
+                "tags": [
+                    "Facility"
+                ],
+                "produces": [
+                    "application/json"
+                ],
+                "security": [],
+                "parameters": [
```

Comment:
There should be a parameter for feeding in a token here, kindly find out how it's done or I can show you later today

Reply:
@Baystef. A token is always attached to the cookies. Hence negating the need. Kindly check your developer tool and navigate to the Applications section and then to cookies

Reply:
Alright, didn't know the cookie works on swagger. It's fine then